src/platform/web/Platform.js

```diff
@@ -82,6 +82,36 @@ async function loadOlmWorker(paths) {
     return olmWorker;
 }
 
+// needed for mobile Safari which does *not* resize the viewport when the keyboard
+// shows or hides (see https://bugs.webkit.org/show_bug.cgi?id=141832)
+function adaptUIOnVisualViewportResize() {
+    if (!window.visualViewport) {
+        return;
+    }
+    window.visualViewport.addEventListener('resize', () => {
+        const sessionView = document.getElementsByClassName('SessionView')[0];
```

Comment:
You can just pass in `this._container` from `Platform` in here and set the custom css property on that as they are inherited, so you don't need the hard-coded selector.